e2e/tests/container_create.go

```diff
@@ -749,6 +749,39 @@ func ContainerCreate(opt *option.Option) {
 				Expect(netDetails.MacAddress).Should(Equal(macAddress))
 			}
 		})
+		It("should create a container with readonly root filesystem", func() {
+			// Define options
+			options.Cmd = []string{"sleep", "Infinity"}
+			options.HostConfig.ReadonlyRootfs = true
+
+			// Create container
+			statusCode, ctr := createContainer(uClient, url, testContainerName, options)
+			Expect(statusCode).Should(Equal(http.StatusCreated))
+			Expect(ctr.ID).ShouldNot(BeEmpty())
+
+			// // Start container
```

Comment:
nit: is this left over? I guess we don't need to actually run the container because we're verifying the contents of the spec?